### DISCUSSION TITLE
Segmentation Fault in WvDial

### DISCUSSION THREAD

**Discussion by MatanzaBR** [void-linux/void-packages#45693] [Q&A]
**Segmentation Fault in WvDial
========================**

**# wvdial**

--> WvDial: Internet dialer version 1.61
--> Initializing modem.
--> Sending: ATZ
OK

...

CONNECT
--> Carrier detected.  Starting PPP immediately.
--> Starting pppd at Sun Aug 20 22:11:49 2023
--> Pid of pppd: 21902
**segmentation fault**

**$ uname -r**
6.4.11_1

$ xbps-query -S wvdial
architecture: x86_64
conf_files:
	/etc/ppp/peers/wvdial
	/etc/ppp/peers/wvdial-pipe
filename-sha256: 670b6d9d516430fb74d03f51b717291de7814322d0ee90685947f8140c0cf5c9
filename-size: 72KB
homepage: https://code.google.com/p/wvstreams/
install-date: 2022-09-25 13:29 -03
installed_size: 190KB
license: LGPL-2.0-only
maintainer: Enno Boland <gottox@voidlinux.org>
metafile-sha256: fdad885aab176434c6c006d56f8ca1316c6fb88cbf1a538a7a197e99e167bf3a
pkgname: wvdial
pkgver: **wvdial-1.61_8**
repository: https://repo-default.voidlinux.org/current
run_depends:
	ppp>=0
	**wvstreams>=4.6.1_20**
	libstdc++>=4.4.0_1
	libgcc>=4.4.0_1
	glibc>=2.32_1
shlib-requires:
	libuniconf.so.4.6
	libwvstreams.so.4.6
	libwvutils.so.4.6
	libwvbase.so.4.6
	libstdc++.so.6
	libgcc_s.so.1
	libc.so.6
short_desc: Point-to-Point Protocol dialer
source-revisions: wvdial:afca7b0266
state: installed

**Comment by classabbyamp** [comment]
if you are still encountering this, please open a [bug report](https://github.com/void-linux/void-packages/issues/new?assignees=&labels=bug%2Cneeds-testing&projects=&template=bug-report.yml)

**Reply by MatanzaBR** [reply]
> if you are still encountering this, please open a [bug report](https://github.com/void-linux/void-packages/issues/new?assignees=&labels=bug%2Cneeds-testing&projects=&template=bug-report.yml)

Is this problem common in other Linux distros or does it only appear in Void Linux?
